cost/aws/unused_rds/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 5.2
+
+- Added `databaseEngine`, `engineVersion`, `platform` and `lookBackPeriod` incident fields
```

Comment:
Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.